docs/notebooks/tutorials/QFM_surface.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<p>missing semicolon.</p><p> </p><p>also this might be better on a log scale?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>